plugins/janus_audiobridge.c

```diff
@@ -2781,6 +2787,28 @@ void janus_audiobridge_incoming_rtp(janus_plugin_session *handle, int video, cha
 		pkt->silence = FALSE;
 		pkt->length = 0;
 
+		/* First check if probation period */
```

Comment:
This probation period is still being done even when FEC is disabled, can you add the check to only do that if FEC is in use? I can do that after the merge otherwise.

Reply:
Whatever you use FEC or not, we can remove probation.

Reply:
I'm not against this probation, since you explained it was actually useful in some cases. I thought this was mostly useful when doing FEC, though: is this not the case?